typo3/cms-core/CVE-2023-47127.yaml

```diff
@@ -0,0 +1,22 @@
+title: 'TYPO3-CORE-SA-2023-006: Weak Authentication in Session Handling'
+link: 'https://typo3.org/security/advisory/typo3-core-sa-2023-006'
+cve: CVE-2023-47127
+branches:
+    8.x:
+        time: '2023-11-14 09:58:00'
+        versions: ['>=8.0.0', '<8.7.55']
+    9.x:
+        time: '2023-11-14 09:58:00'
+        versions: ['>=9.0.0', '<9.5.44']
+    10.x:
+        time: '2023-11-14 09:58:00'
+        versions: ['>=10.0.0', '<10.4.41']
+    11.x:
+        time: '2023-11-14 09:58:00'
+        versions: ['>=11.0.0', '<11.5.33']
+    12.x:
+        time: '2023-11-14 09:58:00'
+        versions: ['>=12.0.0', '<12.4.8']
+    main:
+        time: 
```

Comment:
those two lines should probably be removed as version 13 does not seem to be affected according to the advisory

Reply:
indeed